README.md

```diff
@@ -81,6 +81,24 @@ This will start the process of "ejecting" from Create React Native App's build s
 
 **Warning:** Running eject is a **permanent** action. Please use a version control system, such as git, so you can revert back if necessary. An ejected app will require you to have an [Xcode and/or Android Studio environment](https://facebook.github.io/react-native/docs/getting-started.html) set up.
 
+### Adding Flow
```

Comment:
I think this should probably be in the template User Guide, rather than the repo README